<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>paramsNormalize</class>
 <widget class="QWidget" name="paramsNormalize">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="typeCombo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>Range</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Variance</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Center</string>
    </property>
   </item>
  </widget>
  <widget class="QDoubleSpinBox" name="rangeMinSpin">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="minimum">
    <double>-999999999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>999999998.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="rangeMinLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>80</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Min</string>
   </property>
  </widget>
  <widget class="QLabel" name="rangeMaxLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Max</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="rangeMaxSpin">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="minimum">
    <double>-99999998.000000000000000</double>
   </property>
   <property name="maximum">
    <double>999999986.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Normalize</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="singleDimCheck">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Single Dimension</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="dimensionSpin">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>57</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
